camouflage-flink/src/main/java/com/cldcvr/camouflage/flink/FlinkCamouflageSerDe.java

```diff
@@ -0,0 +1,40 @@
+package com.cldcvr.camouflage.flink;
+
+import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.Objects;
+
+public class FlinkCamouflageSerDe implements Serializable {
```

Comment:
Can we add some java doc comments to mentioned the need of FlinkCamouflageSerDe?

Reply:
It is not a new serde it is same as BeamCamouflageSerDe however it needs to be moved to core which requires to refactor Beam module as well. I havent done that to prevent work of testing.

However yes we need to add a generic doc

Reply:
Tests are in progress